x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/components/providers/selectable.tsx

```diff
@@ -81,11 +90,31 @@ export const SelectableProvider: React.FC<SelectableProviderProps> = ({
   const EuiSelectableContent = useCallback<NonNullable<EuiSelectableProps['children']>>(
     (list, search) => (
       <>
-        {search}
+        <EuiFlexGroup gutterSize="xs" alignItems="center">
+          <EuiFlexItem>{search}</EuiFlexItem>
+          {currentSolution && Object.keys(solutionKeys).includes(currentSolution) ? (
+            <EuiFlexItem grow={false}>
+              <EuiButtonGroup
+                buttonSize="s"
+                legend="Solution filter"
+                idSelected={solutionFilter ?? ''}
+                onChange={(solution) => onSolutionFilterChange(solution as SolutionView)}
+                options={Object.keys(solutionKeys).map((solution) => ({
+                  id: solution,
+                  label: solutionKeys[solution as SolutionView],
```

Comment:
Should the values in `solutionKeys` be internationalized? Since they are being used as labels here?

Reply:
I don't believe we internationalize those ever.